demo-app/src/open-truss/configs/6-state-management-demo.yaml

```diff
@@ -0,0 +1,22 @@
+workflow:
+  version: 1
+  signals:
+    accountIds: AccountIDs
+    userIds: AccountIDs
+  frames:
+    - frame:
+      view:
+        component: IncrementAccountID
+    - frame:
+      view:
+        component: IncrementAccountID
+    - frame:
+      view:
+        component: IncrementAccountID
+        props:
+          accountIds: :userIds
+    - frame:
+      view:
+        component: IncrementAccountID
+        props:
+          accountIds: :userIds
```

Comment:
@kmcq shows alias feature that we talked about. Can see in this demo the two different sets of components that are reading from different signals

https://github.com/open-truss/open-truss/assets/4481584/04da64c3-1078-4380-9981-3967910fbea0

Reply:
So rad! Let's for sure document this ability in the config spec